### PR TITLE
thesist:0.1.0

### PR DESCRIPTION
**Pull request by tfachada**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A Master's thesis template for Instituto Superior Técnico (IST)

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/thesist/0.1.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+/* INIT THESIS */
+// Set language to en/pt
+// Setup title page
+// Hide unused lists
+#show: thesis.with(
+
+  lang: "en",
+
+  cover_image: image("Images/default_cover.jpg", width: 95%),
```
Comment (laurmaedje) [inline packages/preview/thesist/0.1.0/template/main.typ:11]: It is recommended to use kebab-case, as in `cover-image`. It is not a requirement, but it is the common convention in Typst.

Comment (tfachada) [reply packages/preview/thesist/0.1.0/template/main.typ:11]: Ok, will correct. I now have these doubts, though:

- Does that also apply to filenames? If yes, is there an issue with the example files and folders within `template`, which I used a different convention on due to them being on the user side?

- Does that also apply to the `STRING` variables I defined in the `thesis` function in `layout.typ` (which I named to look like `#define` C constants due to their immutability)?

Comment (laurmaedje) [reply packages/preview/thesist/0.1.0/template/main.typ:11]: - With filename, I think there's less of a convention, but I personally would also use kebab-case.
- I haven't seen much all uppercase variables in Typst code anywhere, so I would say it's not common, but I believe the string variables are private anyway, right? Then, it doesn't matter that much.

Comment (tfachada) [reply packages/preview/thesist/0.1.0/template/main.typ]: Ok, done! Let me know if there's anything else.

**Pull request merged**